### ISSUE TITLE
Grabbing work title...

### ISSUE THREAD

**Issue by mcolliver** [mrjpeel/splat#1]
Currently the code grabs the first section of the document title.
If we continue to set work via classroom (which seems sensible) then the documents will have the title like this:

Ben Colliver - HWK5 : tic-tac-toe

So:

- we need to decide on a naming convention (HWKx and CWKx) 
- we need to take off the student name from the start and then grab the 'code' for the work and strip off remainder (ie our title for the work)

**Comment by mrjpeel** [comment]
I agree that the best way forward is to set the HWKx and CWKx would be the best way forward.

**Comment by mcolliver** [comment]
OK - so we now have a function that grabs the code part of a document title....

Thinking a bit more - a student could 'play' with the system by renaming the document....

Perhaps it is better to have the 'document ID' as the key field for each piece of work as this, I presume, will stay the same even if student changes the document title.....
